src/traces/pie/attributes.js

```diff
@@ -79,7 +79,27 @@ module.exports = {
 
     text: {
         valType: 'data_array',
```

Comment:
I agree for the on-graph `textinfo: 'text'` part. But if someone wants to show the same `text` piece in all hover labels? But I guess after this PR, `hovertext` will cover that case.